anxcloud/schema_kubernetes.go

```diff
@@ -0,0 +1,118 @@
+package anxcloud
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+func schemaKubernetesCluster() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Cluster identifier.",
+		},
+		"name": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "Cluster name.",
+			ForceNew:    true,
+		},
+		"version": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Computed:    true,
+			Description: "Kubernetes version.",
+		},
+		"location": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "Cluster location.",
+			ForceNew:    true,
+		},
+		// TODO: should we rename this field in tf? it's actually `needs_service_vms`
```

Comment:
Should we rename this field?

Reply:
The naming is not great, but I think as an attribute, which describes the Cluster resource, `needs_something` is still better than `do_something`...

No matter what, I don't think a 'client' like the TF provider should rename such a field if there is no hard technical reason.

Reply:
Changed it back to `needs_service_vms`